### PR TITLE
rewrite: scraper

### PR DESCRIPTION
**Pull request by TheDivic**
## Description

I rewrote the scraper to scrape all information needed to create links
to jobs on the CircleCI dashboard.

I didn't find a way to link to specific tests, only to jobs those tests
belong to. I couldn't even find a separate test page in the UI, so I think
this is the best we could do.

Also, now it writes output to stdout, but if you need it in a file, you
can easily pipe the output to a file, I think this is a better approach.

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## How Has This Been Tested?

Only manually.

## Checklist:

- [x] I have performed a self-review of this code and spelling
- [x] I have added comments in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: scraper/circleci_scraper.go [hunk 1]
```diff
@@ -74,81 +80,126 @@ func circleciApiCall(url string) []byte {
 	return body
 }
 
-func getPipelines(branch string) Pipelines {
-	var pipelines Pipelines
+func getPipelines(branch string) []Pipeline {
+	body := circleciApiCall(fmt.Sprintf("project/gh/filecoin-project/lotus/pipeline?branch=%s", branch))
```
Comment (TheMenko) [inline scraper/circleci_scraper.go:84]: We want project and repo here besides the branch, since we will use this on different repo's i suppose?

Comment (TheDivic) [reply scraper/circleci_scraper.go:84]: Yep, I'll change it 👍

**Review by TheMenko — APPROVED** [review]

File: frontend/apps/web/src/model/CI.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+// generated from JSON using https://app.quicktype.io/
```
Comment (toljaEmbroker) [inline frontend/apps/web/src/model/CI.ts:1]: Can we do something about the `ci.json` file? There are 158k lines here, I'm not sure how will this impact the performance.

Comment (toljaEmbroker) [reply frontend/apps/web/src/model/CI.ts:1]: hm, e2e-tests time is the same as before, will check it out, maybe it's fine 😄 .

**Review by aleksaToljic — APPROVED** [review]

**Pull request merged**